api-test/formData-k6-dep.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * FormData polyfill for k6
```

Comment:
I would move this file below `api-test/utils` folder

Reply:
ok